ucca/normalization.py

```diff
@@ -108,7 +108,7 @@ def move_sub_scene_elements(node):
 def separate_scenes(node, l1, top_level=False):
     if (node.is_scene() or node.participants) and (top_level or node.parallel_scenes):
         edges = list(node)
-        scene = l1.add_fnode(node, ETags.ParallelScene)
+        scene = l1.add_fnode(node, [(ETags.ParallelScene)])
```

Comment:
Redundant parentheses - if you want a 1-tuple, you must add a comma: `(ETags.ParallelScene,)`. Same for other places in the code